plugin-interface/src/main/scala/mesosphere/marathon/plugin/Group.scala

```diff
@@ -5,7 +5,7 @@ package mesosphere.marathon.plugin
   */
 trait Group {
   def id: PathId
-  def apps: Iterable[RunSpec]
+  def apps: Iterable[(_, RunSpec)]
```

Comment:
whats the first part of the tuple? maybe better to be explicit/a map/a case class?